### DISCUSSION TITLE
umi 打包后的color function （modern）有问题

### DISCUSSION THREAD

**Discussion by chenweiyi** [umijs/umi#8652] [Q&A]
【问题】
某less文件源代码如下：
```
.wrap {
  position: relative;
  min-width: 325px;
  margin-bottom: 16px;
  cursor: pointer;
  background-color: rgba(255 255 255 / 100%);
  border: 1px solid rgb(217 217 217 / 60%);
  border-radius: 3px;
  background-position-y: -10px !important;
}
```
其中background-color , border 属性的rgb() 采用[modern方式](https://stylelint.io/user-guide/rules/list/color-function-notation/) ，但是打包完成后的，在浏览器上看到的样式是如下图所示：
![image](https://user-images.githubusercontent.com/15872986/180115870-94157564-7651-41e4-a31c-bf0cda299761.png)

【现象】
如上图，可以看到打包后，rgb的的写法变成了[legacy方式](https://stylelint.io/user-guide/rules/list/color-function-notation/) ，参数个数不对，由4位变成了3个参数，最后一个参数变成了源代码的最后两个参数相除：
`border: 1px solid rgb(217 217 3.61666667%)`

【期望】
最后两位不相除，保留下来。

**Comment by fz6m** [comment]
新版本 less v4 是支持现代 rgba 的，旧版本 less v3 不支持，你正在使用 umi3 ？

FYI：

 - 可以在这里尝试 https://lesscss.org/less-preview

**Reply by funazhyxi** [reply]
我用的也是less v4 , umijs版本是v3.5.21
![image](https://user-images.githubusercontent.com/109773404/180695305-faf05eec-7880-4b66-9556-5520140b68a7.png)
